operators/multiclusterobservability/controllers/multiclusterobservability/multiclusterobservability_controller.go

```diff
@@ -151,6 +151,28 @@ func (r *MultiClusterObservabilityReconciler) Reconcile(ctx context.Context, req
 		return ctrl.Result{}, nil
 	}
 
+	if _, ok := config.BackupResourceMap[instance.Spec.StorageConfig.MetricObjectStorage.Key]; !ok {
+		config.BackupResourceMap[instance.Spec.StorageConfig.MetricObjectStorage.Key] = config.ResourceTypeSecret
+	}
+
+	if res, ok := config.BackupResourceMap[req.Name]; ok {
+		var err error = nil
+		switch res {
+		case config.ResourceTypeConfigMap:
+			err = util.AddBackupLabelToConfigMap(r.Client, req.Name, config.GetDefaultNamespace())
+		case config.ResourceTypeSecret:
+			err = util.AddBackupLabelToSecret(r.Client, req.Name, config.GetDefaultNamespace())
+		default:
+			// we should never be here
+			log.Info("unknown type " + res)
+		}
+		
+		if err != nil {
+			reqLogger.Error(err, "Failed to add backup label")
+			return ctrl.Result{}, err
+		}
+	}
+
```

Comment:
where is the logic to check whether the resources are the custom dashboard configmaps?